### PR TITLE
add margin to drop

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR adds margin for top and bottom of drop 
#### Should this PR be placed on the NEXT branch (design-system theme)?
yes
#### What testing has been done on this PR?
design system to match figma files
#### Any background context you want to provide?
https://github.com/hpe-design/design-system/issues/657
#### What are the relevant issues?
issue #657 on design system
#### Screenshots (if appropriate)

Screen Shot 2020-08-18 at 12 11 41 PM
Screen Shot 2020-08-18 at 12 11 55 PM
Screen Shot 2020-08-18 at 12 12 22 PM

Screen Shot 2020-08-18 at 12 11 32 PM

#### Is this change backward compatible or could it be a breaking change for the official HPE theme?
backward compatible
#### How should this PR be communicated in the release notes?
This should be added that previously there was no margin between `textinput` and `drop` and now it is 6px

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -186,6 +186,9 @@ export const hpe = deepFreeze({
       border: {
         radius: '4px',
       },
+      extend: ({ theme }) =>
+        `margin-top: ${theme.global.edgeSize.xsmall};
```
Comment (taysea) [inline src/js/index.js:190]: For future reference, `theme.global.edgeSize.xsmall` = `6px` which aligns with what is shown in designs.

File: src/js/index.js [hunk 2]
```diff
@@ -186,6 +186,9 @@ export const hpe = deepFreeze({
       border: {
         radius: '4px',
       },
+      extend: ({ theme }) =>
+        `margin-top: ${theme.global.edgeSize.xsmall}; 
+       margin-bottom: ${theme.global.edgeSize.xsmall};`,
```
Comment (halocline) [inline src/js/index.js:191]: Just a formatting suggestion.

```suggestion
      extend: ({ theme }) => `
        margin-top: ${theme.global.edgeSize.xsmall};
        margin-bottom: ${theme.global.edgeSize.xsmall};
      `,
```

**Comment by britt6612** [comment]
> Realizing the behavior is a little weird for cases like `Menu` where the button overlays when the Menu opens but gets shifted down by the 6px. seems like in that case of menu we wouldn't want this margin to apply.

I. just noticed that as well that you pointed it out. It didnt stand out to me before

**Comment by britt6612** [comment]
Let me check if there is a way I can exclude menu from having that margin

**Comment by britt6612** [comment]
@taysea thought of a great idea with just ignoring menu.
In grommet there is default values for `align-top` and `align-bottom` 
since we know what those are we can check to see if those are set aka if its `menu` and not set the margin

**Pull request merged**